server/channels/store/sqlstore/store.go

```diff
@@ -109,7 +109,6 @@ type SqlStoreStores struct {
 	postPriority               store.PostPriorityStore
 	postAcknowledgement        store.PostAcknowledgementStore
 	postPersistentNotification store.PostPersistentNotificationStore
-	trueUpReview               store.TrueUpReviewStore
```

Comment:
Are there any DB migrations that need to be done to remove this table or recreate it if someone were to downgrade?

Reply:
Good call, added!